src/CsharpClient/Quix.Streams.PerformanceTest/PerformanceTest.cs

```diff
@@ -18,14 +18,14 @@ public void Run(CancellationToken ct)
         {
             var client = new TestStreamingClient(CodecType.ImprovedJson);
 
-            //var topicConsumer = client.CreateTopicConsumer();
-            var topicProducer = client.CreateTopicProducer();
+            //var topicConsumer = client.GetTopicConsumer();
+            var topicProducer = client.GetTopicProducer();
 
             //topicConsumer.OnStreamReceived += (s, stream) =>
             //{
-            //    var buffer = stream.Parameters.CreateBuffer();
+            //    var buffer = stream.Timeseries.CreateBuffer();
```

Comment:
![image](https://user-images.githubusercontent.com/50139826/220993064-7fcc522b-715f-4ccb-9e94-026d6471fe2f.png)
VS 
![image](https://user-images.githubusercontent.com/50139826/220993155-2eef279c-9ee1-47f6-aa64-93dbce80a9f0.png)

Reply:
shouldn't this be lowercase? or is that for python only? I won't repeat this comment but there are more instances

Reply:
c# upper, python lower